<<<<<<< HEAD
#Service to Stop the robot

---
int64 ret
=======
#Service to Stop the robot

---
int64 ret
>>>>>>> 1b1f87b6247333221ca1660d810eda01b82fd03f
string msg